themes/bootstrap3/less/components/search.less

```diff
@@ -97,7 +97,12 @@ header .container.navbar { margin-bottom: 0; }
   .record-checkbox,
   .record-number { display: none; }
 
-  .title { font-weight: bold; }
+  .title,
+  .result-author {
```

Comment:
Do you really want to make the authors bold? It feels very heavy to me.

Reply:
Good catch! 👍